native/cocos/editor-support/spine-wasm/spine-skeleton-instance.cpp

```diff
@@ -579,7 +586,7 @@ void SpineSkeletonInstance::resizeSlotRegion(const spine::String &slotName, uint
         auto *attachmentVertices = static_cast<AttachmentVertices *>(region->getRegion()->rendererObject);
         if (createNew) {
             attachmentVertices = attachmentVertices->copy();
-            region->getRegion()->rendererObject = attachmentVertices;
+            info.attachmentVertices = attachmentVertices;
```

Comment:
Would info.attachmentVertices not be nullptr?

Reply:
Before cache slotInfo to _slotTextureSet, old value will be released firstly.